### DISCUSSION TITLE
Achievement folders

### DISCUSSION THREAD

**Discussion by meleu** [RetroAchievements/RAIntegration#769] [Ideas]
_From @kdecks on April 8, 2018 5:7_

This is probably a hybrid RASuite/RAWeb issue.

This concept is for visual grouping:

Instead of
![image](https://user-images.githubusercontent.com/32706333/38463308-08d28db8-3ab5-11e8-894a-1c4b208af07f.png)
We get
```
+ Penguin Craze
    - Penguin Craze I
    - Penguin Craze II
    - Penguin Craze III
    - Penguin Craze IV
    - Penguin Craze V
    - Penguin Craze VI
```
Where the + designates collapsible folders and - designates contents hidden after being collapsed.
Each one of the Penguin Craze I through VI would all be normal achievements.

#### **Properties:**
The grouping is to allow devs to more clearly show set structure and help visually simplify game pages. The folder is not an achievement, but would display a percentage of the achievements completed within it. The folder would be able to be expanded and contracted on the game page and inside the emulator. Only one level of folders would be allowed, no sub folders. A default folder icon would be used for visual consistency.

_Copied from original issue: RetroAchievements/RASuite#75_

**Comment by salvadorc17** [comment]
Does the sub achievements are linked to main ones with ids? Also the "folders" are only used for site layout

**Comment by Jamiras** [comment]
I'm going to split sub-achievements into a separate issue (#768)
